app/UserGroups.php

```diff
@@ -93,4 +93,13 @@ public function scopeConfirmedInvitation($query)
     {
         return $query->where('status', 1)->orWhereNull('status');
     }
+
+    public function getFullName()
```

Comment:
Is this called anywhere?

Reply:
Yes, but perhaps not anymore. These calls were the cause of the error that appeared when a user was soft deleted:

https://github.com/TheRestartProject/restarters.net/blob/master/resources/views/partials/volunteer-row.blade.php#L30
https://github.com/TheRestartProject/restarters.net/blob/master/resources/views/partials/volunteer-badge.blade.php#L27

Reply:
Ah OK, bit confused as from diff it looks as if it's a completely new method.